PRINCIPLES.md

```diff
@@ -53,4 +53,8 @@ In addition to the principles mentioned above, the Wikibase API also supports he
 
 ## Errors
 
-https://www.mediawiki.org/wiki/Core_Platform_Team/Initiative/Core_REST_API_in_Mediawiki/Design_principles#Errors
\ No newline at end of file
+https://www.mediawiki.org/wiki/Core_Platform_Team/Initiative/Core_REST_API_in_Mediawiki/Design_principles#Errors
+
+## Response bodies upon changes
+
+When resources are returned upon creation or change we do **not** include meta information, like edit tags or the summary. The resource as it is returned may be the result of a merge between what was submitted and server state, so there is no direct connection between the resource state and the submitted payload anymore.
```

Comment:
unclear why we are saying "anymore" here.
I'm not sure where this has ever been the behaviour of our APIs?

Reply:
As in... Normally you'd think you can tell the response from your request, but after a merge this is not true _anymore_.

Will remove it.